.github/workflows/kind.yaml

```diff
@@ -0,0 +1,92 @@
+name: Test DC App in a KinD Cluster
+
+on:
+  workflow_call:
+    inputs:
+      dc_app:
+        required: true
+        type: string
+    secrets:
+      BITBUCKET_LICENSE:
+        description: 'Bitbucket license'
+      CONFLUENCE_LICENSE:
+        description: 'Confluence license'
+      JIRA_LICENSE:
+        description: 'Jira license'
+      BAMBOO_LICENSE:
+        description: 'Bamboo license'
+
+jobs:
+  kind-testing:
+    runs-on: ubuntu-latest
+    env:
+      # See: https://github.com/kubernetes-sigs/kind/tags
+      KIND_VERSION: "v0.17.0"
+      # See: https://hub.docker.com/r/kindest/node/tags
+      K8S_VERSION: "v1.25.3"
+      DC_APP: ${{inputs.dc_app}}
+      LICENSE: ${{ secrets[format('{0}_LICENSE', inputs.dc_app)] }}
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@master
+
+      - name: Pin Kubectl version
+        uses: azure/setup-kubectl@v3.0
+        with:
+          version: 'v1.25.3'
+
+      - name: Pin Helm version
+        uses: azure/setup-helm@v3
+        with:
+          version: v3.9.2
+
+      - name: Create KinD cluster
```

Comment:
I like that they are distinct scripts in the workflow, so it should be easier to read the results e.g. when failure happens.